data/campaigns/Heir_To_The_Throne/scenarios/29_Digging_for_Dwarves.cfg

```diff
@@ -0,0 +1,599 @@
+                side=2
+                caption=""
+                image=none  # can't use narrator, since Konrad doesn't exist yet
+                message=_"<i>The Knalgans led Konrad and his men, blindfolded, through a maze of twisting tunnels and passageways.
+
+The sound of hammers echoed through the stale cave air, then pickaxes, then the quiet murmur of low dwarvish voices, interspersed with louder remarks from the still-seeing dwarves of Konrad’s own party. And at long last, silence.</i>"
+            [/message]
+        )} {ELSE(
+            [message]
+                side=2
+                caption=""
+                image=none  # can't use narrator, since Konrad doesn't exist yet
+                message=_"<i>The Knalgans led Konrad and his men, blindfolded, through a maze of twisting tunnels and passageways.
+
+The sound of hammers echoed through the stale cave air, then pickaxes, then the quiet murmur of low dwarvish voices. And at long last, silence.</i>"
+            [/message]
+        )} {/IF}
+
+        #############################
+        # KONRAD ARRIVES
+        #############################
+        {DELAY 500}
+        {NAMED_UNIT 5 "Dwarvish Stalwart" 1 6 Geldar _"Geldar" ()} {FACING se} {ANIMATE} {REDRAW} # must be side 1, so that the player can see
+        {MOVE_UNIT id=Geldar 5 7}
+        {RECALL_XY Konrad 1 6} {MODIFY_UNIT id=Konrad facing se}  {REDRAW}
+        {MOVE_UNIT id=Konrad 4 7}
+        [message]
+            speaker=Geldar
+            message=_"Here we are. The start o’ the cave-in. Somewhere beyond here, past all manner of beasts and vermin, lie our trapped miners. Rescue them, and you’ll be rewarded."
+        [/message]
+        [message]
+            speaker=Konrad {KONRAD_VARIATION concerned}
+            message=_"It’s just a solid wall of rock. How am I supposed to get through and find your comrades?"
+        [/message]
+        [message]
+            speaker=Geldar
+            message=_"Dig, human. I must return to my guard duties."
+        [/message]
+        {MOVE_UNIT id=Geldar 1 6}
+        {KILL id=Geldar}
+        {RECALL_KONRAD_AND_COMPANIONS 48 4}
+
+        #############################
+        # OBJECTIVES
+        #############################
+        [objectives]
+            [objective]
+                description=_"Find the missing dwarves"
+                condition=win
+            [/objective]
+            {OBJECTIVES_HERODEATHS}
+            {TURNS_RUN_OUT}
+            [gold_carryover]
+                carryover_percentage=40
+                bonus=yes
+            [/gold_carryover]
+        [/objectives]
+    [/event]
+
+    #############################
+    # FORESHADOW WITNESS
+    #############################
+    [event]
+        name=turn 4
+        {IF} {HAVE_UNIT id=Ulfdain} {THEN(
+            [message]
+                speaker=Konrad
+                message=_"Ulfdain, the Knalgans here have very evasive about what’s going on. Do you have any insights?"
```

Comment:
I don't know what "have very evasive" means here. (same thing in the else case)

Reply:
I think it was supposed to be 'have **been** very evasive'. Clearly I missed this one, well spotted.